.env

```diff
@@ -0,0 +1,8 @@
+MYSQLDB_DATABASE=bookStore
+MYSQLDB_ROOT_USER=root
+MYSQLDB_ROOT_PASSWORD=156354OlEh
```

Comment:
we do not need this file for **Add Category model** task

Reply:
because i i did the docker task first.